lib/rubocop/sketchup/tool_checker.rb

```diff
@@ -13,11 +13,9 @@ def on_class(node)
 
       private
 
-      # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
Ah, RuboCop finally understand abstract methods that throws `NotImplementedError`?

Reply:
Fixes, new Cops, etc, etc...